pkg/cmd/market.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/c9s/bbgo/pkg/bbgo"
+)
+
+func init() {
+	marketCmd.Flags().String("session", "", "the exchange session name for querying information")
+	RootCmd.AddCommand(marketCmd)
+}
+
+// go run ./cmd/bbgo market --session=ftx --config=config/bbgo.yaml
+var marketCmd = &cobra.Command{
+	Use:          "market",
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := context.Background()
+
+		configFile, err := cmd.Flags().GetString("config")
+		if err != nil {
+			return err
+		}
+
+		if len(configFile) == 0 {
+			return errors.New("--config option is required")
+		}
+
+		if _, err := os.Stat(configFile); os.IsNotExist(err) {
+			return err
+		}
+
+		userConfig, err := bbgo.Load(configFile, false)
+		if err != nil {
+			return err
+		}
+
+		environ := bbgo.NewEnvironment()
+		if err := environ.ConfigureDatabase(ctx); err != nil {
+			return err
+		}
+
+		if err := environ.ConfigureExchangeSessions(userConfig); err != nil {
+			return err
+		}
+
+		sessionName, err := cmd.Flags().GetString("session")
+		if err != nil {
+			return err
+		}
+
+		session, ok := environ.Session(sessionName)
+		if !ok {
+			return fmt.Errorf("session %s not found", sessionName)
+		}
+
+		markets, err := session.Exchange.QueryMarkets(ctx)
+		if err != nil {
+			return err
+		}
+
+		log.Infof("markets: %+v", markets)
```

Comment:
iterate the markets and print the market one by one? so that the format could be easier for reading.